src/cntk-train/src/main/scala/CNTKLearner.scala

```diff
@@ -154,9 +154,10 @@ class CNTKLearner(override val uid: String) extends Estimator[CNTKModel] with CN
     val output = cb.runCommand
     logInfo("CNTK training finished")
     conformedData.unpersist()
+    val modelPath = "file://" + config.getModelPath
```

Comment:
+1. Can't we make config.getModelPath return the correct URI (local vs. HDFS vs. WASB etc)